Assets/Shared/Scripts/SceneMachine.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public static class SceneMachine {
+
+	public static void LoadTheScene(string scene){
```

Comment:
I think if we still have this class, let's make Doorway use it. If Doorway is going to call SceneManager directly, we can kill this class for now.